docs/source/loading_datasets.rst

```diff
@@ -158,22 +158,52 @@ Apart from :obj:`name` and :obj:`split`, the :func:`datasets.load_dataset` metho
 
+From a community dataset on the Hugging Face Hub
+-----------------------------------------------------------
+
+The community shares hundreds of datasets on the Hugging Face Hub using **dataset repositories**.
+A dataset repository is a versioned repository of data files.
+Everyone can create a dataset repository on the Hugging Face Hub and upload their data.
+
+For example we have created a demo dataset at https://huggingface.co/datasets/lhoestq/demo1.
+In this dataset repository we uploaded some csv files, and you can load the dataset with:
+
+.. code-block::
+
+    >>> from datasets import load_dataset
+    >>> dataset = load_dataset('lhoestq/demo1')
+
+You can even choose which files to use in a dataset repository.
+For example you can load a subset of the **C4 dataset for language modeling**, hosted by AllenAI on the Hub.
+You can browse the dataset repository at https://huggingface.co/datasets/allenai/c4
+
+In the following example we specify which subset of the files to use with the ``data_files`` parameter:
+
+.. code-block::
+
+    >>> from datasets import load_dataset
+    >>> c4_subset = load_dataset('allenai/c4', data_files='en/c4-train.0000*-of-01024.json.gz')
+
+It will use all the files that match the pattern passed in ``data_files``.
+If you don't specify which data files to use, it will use all the data files (here all C4 is about 13TB of data).
+
 
 .. _loading-from-local-files:
 
-From local files
+From local or remote files
 -----------------------------------------------------------
 
-It's also possible to create a dataset from local files.
+It's also possible to create a dataset from your own local or remote files.
 
 Generic loading scripts are provided for:
 
 - CSV files (with the :obj:`csv` script),
 - JSON files (with the :obj:`json` script),
 - text files (read as a line-by-line dataset with the :obj:`text` script),
+- parquet files (with the :obj:`parquet` script).
 - pandas pickled dataframe (with the :obj:`pandas` script).
 
-If you want to control better how your files are loaded, or if you have a file format exactly reproducing the file format for one of the datasets provided on the `HuggingFace Hub <https://huggingface.co/datasets>`__, it can be more flexible and simpler to create **your own loading script**, from scratch or by adapting one of the provided loading scripts. In this case, please go check the :doc:`add_dataset` chapter.
+If you want to control better how your files are loaded, or if you have a file format exactly reproducing the file format for one of the datasets provided on the `Hugging Face Hub <https://huggingface.co/datasets>`__, it can be more flexible and simpler to create **your own loading script**, from scratch or by adapting one of the provided loading scripts. In this case, please go check the :doc:`add_dataset` chapter.
```

Comment:
```suggestion
If you want more fine-grained control on how your files are loaded or if you have a file format that matches the format for one of the datasets provided on the `Hugging Face Hub <https://huggingface.co/datasets>`__, it can be more  simpler to create **your own loading script**, from scratch or by adapting one of the provided loading scripts. In this case, please go check the :doc:`add_dataset` section.
```